packages/yargs/mixins/main/mixin.core.js

```diff
@@ -37,10 +35,10 @@ class YargsMixin extends Mixin {
       (duplicate) => `package '${duplicate}' should be installed just once`
     );
   }
-  handleError(error) {
+  handleError(error, recoverable) {
     // eslint-disable-next-line no-console
     console.error(error.stack ? error.stack.toString() : error.toString());
```

Comment:
Why don't we use the logger here, too?

Reply:
Because this is just the default/fallback implementation - the log mixin adds a proper one.